### PR TITLE
Add type definitions for the k6 experimental csv module

### PR DESCRIPTION
**Pull request by oleiade**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: [documentation PR](https://github.com/grafana/k6-docs/pull/1724/files#diff-fa475f988461af4e1482dae6b674dac8d0db80bc358979705cb6d5f079655248)
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Review by olegbespalov — COMMENTED** [review]
@oleiade, the PR looks good to me, but since it should be merged to the release branch [release-v0.54.0](https://github.com/grafana/k6-DefinitelyTyped/tree/release-v0.54.0) (I've changed the PR's target) it has a conflict. Please resolve it, and we can merge it there :+1:

**Comment by oleiade** [comment]
Rebased and conflicts solved 🚀

File: types/k6/tsconfig.json [hunk 1]
```diff
@@ -32,6 +32,8 @@
         "crypto/index.d.ts",
         "browser/index.d.ts",
 
+        "experimental/browser/index.d.ts",
```
Comment (inancgumus) [inline types/k6/tsconfig.json:35]: We removed the experimental browser module in [this PR](https://github.com/grafana/k6-DefinitelyTyped/pull/63).

File: types/k6/tsconfig.json [hunk 2]
```diff
@@ -42,6 +44,8 @@
         "test/data.ts",
         "test/encoding.ts",
         "test/execution.ts",
+        "test/experimental/browser.ts",
```
Comment (inancgumus) [inline types/k6/tsconfig.json:47]: Also here.

**Review by inancgumus — CHANGES_REQUESTED** [review]
LGTM, but we need to remove the experimental module references from this PR :)

File: types/k6/tsconfig.json (hunk 1 above)
Comment (oleiade) [reply types/k6/tsconfig.json:35]: Sorry for the rebase mishap, fixing it 👍🏻

File: types/k6/tsconfig.json (hunk 2 above)
Comment (oleiade) [reply types/k6/tsconfig.json:47]: Sorry for the rebase mishap, fixing it 👍🏻

**Review by inancgumus — APPROVED** [review]
We're now good 👍 Thanks!

**Review by olegbespalov — APPROVED** [review]

**Pull request merged**